Sources/FormattingHelpers.swift

```diff
@@ -1651,6 +1651,24 @@ extension Formatter {
 
 /// Helpers for recursively traversing the declaration hierarchy
 extension Formatter {
+    /// Applies `forEachRecursiveDeclarations` in place
+    func forEachRecursiveDeclarations(_ operation: (Declaration) -> Void) {
```

Comment:
A singular name works better here, e.g. "perform operation for each recursive declaration" rather than "perform operation for each recursive declarations"

```suggestion
    func forEachRecursiveDeclaration(_ operation: (Declaration) -> Void) {
```